scripts/changelog.js

```diff
@@ -688,7 +688,7 @@ function dedupeEntries(changelog, items) {
 // (with format `@username`) of everyone who contributed to this version.
 function extractContributors(entries) {
 	const set = Object.values(entries).reduce((memo, pullRequest) => {
-		if (pullRequest.author.login !== "greenkeeper" && pullRequest.author.login !== "renovate-bot") {
+		if (pullRequest.author.login !== "renovate") {
```

Comment:
```suggestion
		if (pullRequest.author.login !== "renovate-bot") {
```

Reply:
> So we want to filter to `renovate-bot`

That doesn't work, `renovate` is what I get from the API:

<img width="734" alt="screen shot 2019-01-29 at 01 11 03" src="https://user-images.githubusercontent.com/113730/51887751-d17e6700-2362-11e9-8057-287618dacfc0.png">

With `pullRequest.author.login !== "renovate-bot"`:

<img width="547" alt="screen shot 2019-01-29 at 00 57 08" src="https://user-images.githubusercontent.com/113730/51887759-d93e0b80-2362-11e9-9f83-5bb058d738a2.png">

With `pullRequest.author.login !== "renovate"`:

<img width="548" alt="screen shot 2019-01-29 at 00 57 29" src="https://user-images.githubusercontent.com/113730/51887769-e0fdb000-2362-11e9-9372-447c1958bbe0.png">

Reply:
Try querying `type`, maybe it will tell you its a bot:
> "type": "Bot",

Reply:
Yeah, went with `__typename` on PRs. For commits, there is no straight way to differentiate (not that it's going to be an issue, but for completeness), so I added the heuristic that it ends with `-bot`.
Really not a big deal on commits because we currently do not have bots that make commits directly in master 🤷‍♂️